modules/programs/superfile.nix

```diff
@@ -142,7 +143,9 @@ in
       ];
     in
     mkIf cfg.enable {
-      home.packages = mkIf (cfg.package != null) [ cfg.package ];
+      home.packages = mkIf (cfg.package != null) (
+        [ cfg.package ] ++ optional (cfg.settings ? metadata && cfg.settings.metadata) pkgs.exiftool
```

Comment:
Should make this supported by a nullable package option to give user's more control.